mws/resource_mws_ncc_binding.go

```diff
@@ -0,0 +1,52 @@
+package mws
+
+import (
+	"context"
+	"log"
+
+	"github.com/databricks/databricks-sdk-go/service/provisioning"
+	"github.com/databricks/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceMwsNccBinding() common.Resource {
+	type binding struct {
+		WorkspaceId int64  `json:"workspace_id" tf:"force_new"`
+		NccId       string `json:"network_connectivity_config_id"`
+	}
+	s := common.StructToSchema(binding{}, common.NoCustomize)
+	p := common.NewPairSeparatedID("workspace_id", "network_connectivity_config_id", "/")
+	createOrUpdate := func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
+		acc, err := c.AccountClient()
+		if err != nil {
+			return err
+		}
+		wait, err := acc.Workspaces.Update(ctx, provisioning.UpdateWorkspaceRequest{
+			NetworkConnectivityConfigId: d.Get("network_connectivity_config_id").(string),
+			WorkspaceId:                 int64(d.Get("workspace_id").(int)),
+		})
+		if err != nil {
+			return err
+		}
+		_, err = wait.Get()
+		if err != nil {
+			return err
+		}
+		p.Pack(d)
+		return nil
+	}
+	return common.Resource{
```

Comment:
Should we somehome specify that import isn't supported?

Reply:
Should we do this in readme docs?